AGENT_CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Datadog Agent version [7.38.2](https://github.com/DataDog/datadog-agent/blob/master/CHANGELOG.rst#7382)
```

Comment:
Was there supposed to be anything for 7.38.1?

Reply:
No changes afaict